scenarios/kubernetes_e2e.py

```diff
@@ -434,6 +434,12 @@ def main(args):
       'KUBE_GKE_NETWORK=%s' % cluster,
     )
 
+    if args.logexporter == 'true':
+        mode.add_environment(
+            'ENABLE_LOGEXPORTER=true',
+            'GCS_ARTIFACTS_DIR=%s' % args.gcs_artifacts_dir,
```

Comment:
This is similar to what we're doing currently for other env vars too (which are used by cluster/log-dump.sh script when we shell out to it from kubetest). This has the advantage that we can just run the logexporter from within the container anytime we want (even in between the test (which can be useful for long tests on large clsuters)) as we have the right parameters available through env vars. Does that SGTY?